rpc/cloudagent/compute/galleryimage/moc_cloudagent_galleryimage.proto

```diff
@@ -65,6 +65,7 @@ message GalleryImage {
 	ImageSource sourceType = 13;
 	HyperVGeneration hyperVGeneration = 14;
 	CloudInitDataSource cloudInitDataSource = 15;
+	int64 size = 16;
```

Comment:
The name should include the unit as well. sizeInBytes?

Reply:
Since node agent vhd has it as size only too, I thought it might be better to align them. Do you still prefer sizeInBytes?

Reply:
Yes